supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/statuspolling/JesPollingActor.scala

```diff
@@ -141,11 +107,9 @@ object JesPollingActor {
     * Forcing the minimum value to be 1 second, for now it seems unlikely to matter and it makes testing a bit
     * easier
     */
-  def determineBatchInterval(qps: Int): FiniteDuration = {
-    val maxInterval = MaxBatchSize / qps.toDouble // Force this to be floating point in case the value is < 1
+  def determineBatchInterval(qps: PositiveInt): FiniteDuration = {
+    val maxInterval = MaxBatchSize / qps.value.toDouble // Force this to be floating point in case the value is 0
```

Comment:
Is this comment still relevant? `qps: PositiveInt` can never be zero. But perhaps one still wants `maxInterval` to be a double.